trunk/scripts/verify_confs.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+TRUNK_DIR=$(dirname -- "${BASH_SOURCE[0]}")/../
```

Comment:
Test file? Should remove it.

Reply:
No, the var store the path of `trunk` folder, the almost the source root of srs project, then doing the next steps.
run `srs -t -c [conf file]` for every file inside `${TRUNK_DIR}/conf/*`.